ui/playTurn/playTurn.component.html

```diff
@@ -1,12 +1,18 @@
 <div>
   <div class="panel panel-default">
-    <div class="panel-heading">Playing Turn: {{playTurnState.game.displayName}}</div>
+    <div class="panel-heading">Playing Turn: <a href="javascript:void(0)" (click)="openGameOnWeb()">{{playTurnState.game.displayName}}</a></div>
     <div class="panel-body">
       <h3>{{status}}</h3>
-      <progressbar *ngIf="curBytes" [animate]="false" [max]="maxBytes" [value]="curBytes"><span style="color:white;white-space:nowrap;">{{curBytes}} K / {{maxBytes}} K</span></progressbar>
-      <button *ngIf="saveFileToUpload" type="submit" class="btn btn-primary btn-sm btn-block" (click)="submitFile(game)">{{ abort ? "Re-" : "" }}Submit File</button>
-      <button *ngIf="saveFileToUpload" type="submit" class="btn btn-default btn-sm btn-block" (click)="watchForSave()">{{ abort ? "Watch for New Save" : "Ignore" }}</button>
-      <button *ngIf="abort || downloaded" type="submit" class="btn btn-default btn-sm btn-block" (click)="goHome()">Return to Game List</button>
+      <div style="height: 35px;">
+        <progressbar *ngIf="curBytes" [animate]="false" [max]="maxBytes" [value]="curBytes"><span style="color:white;white-space:nowrap;">{{curBytes}} K / {{maxBytes}} K</span></progressbar>
+        <button *ngIf="saveFileToUpload" type="submit" class="btn btn-primary btn-sm btn-block" (click)="submitFile(game)">{{ abort ? "Re-" : "" }}Submit File</button>
+        <button *ngIf="saveFileToUpload" type="submit" class="btn btn-default btn-sm btn-block" (click)="watchForSave()">{{ abort ? "Watch for New Save" : "Ignore" }}</button>
+        <button *ngIf="abort || downloaded" type="submit" class="btn btn-default btn-sm btn-block" (click)="goHome()">Return to Game List</button>
+      </div>
+      <div style="padding-top: 10px;">
+        <pydt-game-players [game]="playTurnState.game" [gamePlayerProfiles]="playTurnState.gamePlayerProfiles"></pydt-game-players>
+        <markdown>{{playTurnState.game.description}}</markdown>
```

Comment:
I think the code here should probably be `<markdown [data]="playTurnState.game.description"></markdown>`, otherwise the markdown syntax might not actually get rendered

Reply:
i fixed it. but just now while using the dev client to play games, discovered that external links open in the client and not to default browser. since there is no right click, cannot navigate back to the game list.

Reply:
Yeah, you'd have to intercept the link clicks somehow and use `app.ipcRenderer.send('opn-url', ...the url...)` to get them to open outside of the client.